### PR TITLE
Added Ludicrous mode, Updated Certs and added docker changes

### PR DESCRIPTION
**Pull request by kaushik691**
Currently works with master branch, need to check with releases.
run:
python3 env_setup.py -l All -c enabled | disabled
 python3 create_docker_yml.py -c enabled | disabled 
-> it includes even docker file setup based on configurations.

### CODE REVIEW

**Review by souravDgraph — APPROVED** [review]

File: test_suites/dgraph/Linux/dgraph_suite_03.robot [hunk 1]
```diff
@@ -0,0 +1,54 @@
+*** Settings ***
+Documentation     Dgraph Live Loading Test Suite with ludicrous mode
+Suite Setup        Start Dgraph Ludicrous Mode    local
+Suite Teardown     End All Process    true
```
Comment (souravDgraph) [inline test_suites/dgraph/Linux/dgraph_suite_03.robot:4]: Does true here deletes the p w and ZW dir? if so can we make it false.
We would like to keep them to report bugs.

File: utilities/create_docker_yml.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
+import getopt
+import sys
+import logging
+
```
Comment (souravDgraph) [inline utilities/create_docker_yml.py:4]: line space can be removed.

File: utilities/create_docker_yml.py [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        EX:     
+        python/python3 create_docker_yml.py -c enabled
+    """
+    print(usage_text)
+
+
+def create_docker_compose_file(zeros: int = 1, alphas: int = 1, is_configured: bool = False):
+    services = {}
+    zero_services = {}
+    zero_command = 'dgraph zero --my=zero0:5080 --logtostderr -v=2  '
```
Comment (souravDgraph) [inline utilities/create_docker_yml.py:73]: extra space at end.

**Review by souravDgraph — APPROVED** [review]
Some minor comments to be addressed before push.

File: test_suites/dgraph/Linux/dgraph_suite_03.robot (hunk 1 above)
Comment (kaushik691) [reply test_suites/dgraph/Linux/dgraph_suite_03.robot:4]: okay

File: utilities/create_docker_yml.py (hunk 2 above)
Comment (kaushik691) [reply utilities/create_docker_yml.py:4]: okay

File: utilities/create_docker_yml.py (hunk 3 above)
Comment (kaushik691) [reply utilities/create_docker_yml.py:73]: that's for the command to get appended.

**Pull request merged**